robyn/reloader.py

```diff
@@ -115,11 +117,20 @@ def reload(self):
         clean_rust_binaries(self.built_rust_binaries)
         self.built_rust_binaries = compile_rust_files(self.directory_path)
 
-        self.process = subprocess.Popen(
-            [sys.executable, *arguments],
-            env=new_env,
-            start_new_session=False,
-        )
+        if self.config.dev and self.config.file_path is not None:
```

Comment:
Can we not use `self.file_path`? 

Then we don't need to pass in `config`.

Reply:
we also need `config.dev`. should I pass it as a variable?